### PR TITLE
docs(feedback): Update monitoring page with more Bonita intelligence 

### PR DESCRIPTION
**Pull request by vbotnari12**
based on this jira ticket https://bonitasoft.atlassian.net/browse/DEV-259

### CODE REVIEW

File: modules/ROOT/pages/monitoring.adoc [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
+- An URL should be provided per node
+- The "Connect" button allows to establish and check connection's state
+====
+
+Once the configuration is done, the monitoring page shows a set of metrics for each node.
 
+=== Monitoring single node
 
-== Monitoring a cluster
+image::architecture-single.png[Bonita Central with single node]
```
Comment (jeromecambon) [inline modules/ROOT/pages/monitoring.adoc]: Note sure we want to duplicate the diagram here

File: modules/ROOT/pages/monitoring.adoc [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
+A cluster is a collection of nodes, connected to a load balancer and each running a Bonita Runtime.
+Bonita Central currently uses the load balancer to access one of the nodes - non-determinist to perform the agent setup by using Bonita authentication. 
+Since the application has been connected through its load balancer, Bonita Central may need connection information for each node. 
 See xref:ROOT:network-configuration.adoc[network configuration] for details. +
-Since the application has been connected through its load balancer, Bonita Central may need connection information for each node.
 
-Bonita Central try to connect to each node using usual port and paths.
-If it does not succeed (typically if the port is not the default one), a warning is raised.
-Then, the connections can be configured by clicking on the Configure button:
+image::architecture-cluster.png[Bonita Central with cluster]
```
Comment (jeromecambon) [inline modules/ROOT/pages/monitoring.adoc]: Note sure we want to duplicate the diagram here

File: modules/ROOT/pages/monitoring.adoc [hunk 3]
```diff
@@ -1,27 +1,46 @@
...
+== Monitoring values 
 
-Once the configuration is done, the monitoring page shows a set of metrics for each node.
+Bonita Central will retrieve the monitoring information from the monitoring endpoint by calling each and every active node. 
+Bonita-centered monitoring information, that will help you supervise your Bonita applications. 
+
+- *Work Queue* : Number of *pending works* versus maximum size of the *worker queue*
+- *Connector Queue* : Number of *pending connectors* versus maximum size of the *connector queue*
+- HTTP requests in error
+
```
Comment (jeromecambon) [inline modules/ROOT/pages/monitoring.adoc:39]: I think we need to provide more information here, mention all the metrics, and provide some info on how to  fix eventual issues.

**Review by jeromecambon — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/monitoring.adoc (hunk 3 above)
Comment (vbotnari12) [reply modules/ROOT/pages/monitoring.adoc:39]: That's something I dont master... :(

**Review by Yves-Jacq — APPROVED** [review]

**Review by jeromecambon — APPROVED** [review]

**Pull request merged**